packages/frontend/components/composed/issue/PrevNextIssue/PrevNextIssue.styles.ts

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+import { pxToRem } from "@wdp/lib/theme/functions";
+import { Button } from "components/atomic";
+import { aButton } from "theme/mixins/appearance";
+
+export const Inner = styled.div`
+  padding-block: var(--container-padding-lg);
```

Comment:
Safari doesn't recognize this as a property - for right now, `padding-block-start` and `padding-block-end` need to be defined. Same goes for any padding or margin inline. :(

Reply:
Oh - I think Dana came up with a mixin to solve for this in another project, maybe OFB?